clouditor-engine-core/src/main/java/io/clouditor/assurance/Rule.java

```diff
@@ -83,14 +95,16 @@ public void setCondition(Condition condition) {
 
   public String getAssetType() {
     // single condition
-    if (this.condition != null) {
-      return this.condition.getAssetType();
+    if (this.condition != null && this.condition.getAssetType() != null) {
+      return this.condition.getAssetType().getField();
     }
 
     // multiple conditions
-    if (this.conditions != null && !this.conditions.isEmpty()) {
+    if (this.conditions != null
+        && !this.conditions.isEmpty()
+        && this.conditions.get(0).getAssetType() != null) {
       // take the first one
-      return this.conditions.get(0).getAssetType();
+      return this.conditions.get(0).getAssetType().getField();
```

Comment:
This creates a problem since only the first condition will be evaluated regarding its applicability but in future we are reworking this anyway in #62